src/components/navigation-header/components/product-page-content/utils/types.ts

```diff
@@ -0,0 +1,17 @@
+import { ProductSlug } from 'types/products'
+import { NavigationHeaderItem } from 'components/navigation-header/types'
+
+export interface NavItemLink {
+	type: 'link'
+	label: string
+	url: string
+}
+
+export interface NavItemSubmenu {
+	type: 'submenu'
+	label: string
+	iconColorTheme: ProductSlug
+	items: NavigationHeaderItem[]
+}
```

Comment:
These types are already defined in `PrimaryNavSubmenu` and `PrimaryNavLink`. I think they should be referenced here as the source of truth.

Reply:
Sounds good - have updated this file to reference those types 👍 Let me know if that works!